### ISSUE TITLE
pthread sent an error! undefined:undefined: undefined

### ISSUE THREAD

**Issue by ArneVogel** [niklasf/stockfish.wasm#22]
I am getting the error in the title in firefox 80.0.1. The same code is working in chromium 85.0.4183.83.

Code: https://github.com/ArneVogel/listudy/blob/stockfish_play/lib/listudy_web/templates/study/play.html.eex#L13

Output:
```
hi
pthread sent an error! undefined:undefined: undefined              stockfish.js:45:439
```
`js/stockfish.js  js/stockfish.wasm  js/stockfish.worker.js` were copied from `node_modules/stockfish.wasm/`, which was installed with `npm i stockfish.wasm`.

**Comment by niklasf** [comment]
Is this already hosted anywhere, so I can try it in Firefox without installing the entire thing?

**Comment by ArneVogel** [comment]
Ok I set it up at https://mail.arnevogel.com/en/studies/play but I noticed some other weird behavior. 

It only works in chrome when I dont send the cross-origin-embedder-policy header. But then I get the error `Uncaught ReferenceError: SharedArrayBuffer is not defined` in firefox. 

When I send both headers chrome does not download the file `https://mail.arnevogel.com/js/stockfish.worker.js` with the status: `net::ERR_FAILED`. But firefox doesnt get the SharedArrayBuffer error anymore.

For some reason I don't get the original error in firefox anymore but it does not print anything besides the "hi" like it did in chrome when it worked.

The server is currently sending both headers.
